packages/node_modules/@webex/plugin-meetings/src/meeting/index.js

```diff
@@ -5868,4 +5882,53 @@ export default class Meeting extends StatelessWebexPlugin {
       failure: `${LOG_HEADER} disable bnr failure, `
     });
   }
+
+  /**
+   * starts keepAlives being sent
+   * @returns {void}
+   * @private
+   * @memberof Meeting
+   */
+  startKeepAlive = () => {
+    if (this.keepAliveTimerId) {
+      LoggerProxy.logger.error('Meeting:index#startKeepAlive --> keepAliveTimerId already exists');
```

Comment:
probably all these logs shouldn't be errors, warnings at best as they don't prevent you from joining the call

Reply:
done